jibri/rootfs/defaults/jibri.conf

```diff
@@ -1,6 +1,7 @@
 {{ $JIBRI_USAGE_TIMEOUT := .Env.JIBRI_USAGE_TIMEOUT | default "0" -}}
 {{ $JIBRI_RECORDING_RESOLUTION := .Env.JIBRI_RECORDING_RESOLUTION | default "1280x720" -}}
 {{ $XMPP_TRUST_ALL_CERTS := .Env.XMPP_TRUST_ALL_CERTS | default "true" | toBool -}}
+{{ $ENABLE_XMPP_PORT := .Env.XMPP_PORT | default "0" | toBool }}
```

Comment:
Why not use a single variable and use | default 5222 instead?

Reply:
Because I wanted to leave the default behavior and only specify a value explicitly if the variable is set. This way I'm not forcing the override to the default value but instead only providing the override when needed

Reply:
I don't see any value in that, and that's not how other options are implemented. Being explicit allows one to know exactly what the actual setting is.

Reply:
done